Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj

```diff
@@ -18,8 +18,7 @@
   </ItemGroup>
  
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Converters\Rhino\Speckle.Converters.Rhino7\Speckle.Converters.Rhino7.csproj" />
-    <ProjectReference Include="..\..\..\Sdk\Speckle.Connectors.Common\Speckle.Connectors.Common.csproj" />
+    <ProjectReference Include="..\..\..\Connectors\Rhino\Speckle.Connectors.Grasshopper7\Speckle.Connectors.Grasshopper7.csproj" />
```

Comment:
wait, so Rhino without Grasshopper isn't a thing?

Reply:
Correct, this is the way we did it in v2. While Rhino doesn't rely on the grasshopper code. we want to make sure it is picking dlls that don't conflict in anyway.

Seeing as we'll always be shipping rhino with grasshopper, this was an easy option. But if you can think of alternatives, happy to consider.